recipes/ford/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ford" %}
+{% set version = "6.1.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/FORD-{{ version }}.tar.gz
+  sha256: 24c5868b02a584464c74acfa48b673a8a27ff84c7dc9beda5a37acd4cd3671e0
+
+build:
+  number: 0
```

Comment:
```suggestion
  number: 0
  noarch: python
```

Reply:
What about `importlib-metadata`? This needs a Python version selector.

Reply:
You could remove the selector, installing it on py>=38 should be a no-op and `noarch` packages are generally easier to maintain.